.travis.yml

```diff
@@ -2,6 +2,11 @@ sudo: false
 language: go
 go:
   - 1.8.x
+  - master
```

Comment:
Yes, it's the latest way of specifying it in Travis, that's why I changed it.

Reference: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use